app/Http/Controllers/Users/AdminController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers\Users;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\Users\Admin;
+
+class AdminController extends Controller
+{
+    private $admin;
+
+    public function __construct(Admin $admin)
+    {
```

Comment:
![image](https://github.com/kredo-school/18tha-gj_mall/assets/62535075/8bfcec63-3524-4f6e-bf01-fa8ea87f88e7)

This part of the code is not being used. You may want to remove it and just add it when you use it.